src/test/java/octo/waddle/hellodropwizard/api/MessageTest.java

```diff
@@ -0,0 +1,23 @@
+package octo.waddle.hellodropwizard.api;
+
+import static io.dropwizard.testing.FixtureHelpers.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import io.dropwizard.jackson.Jackson;
+import org.junit.Test;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class MessageTest {
+
+    private static final ObjectMapper MAPPER = Jackson.newObjectMapper();
+
+    @Test
+    public void serializesToJSON() throws Exception {
+        final Message msg = new Message("Hello, Tester!");
+
+        final String expected = MAPPER.writeValueAsString(
```

Comment:
To nie jest test Message'a
Testujesz tutaj serializacje
Jedyny test jednostkowy jaki tutaj możesz napisać to:
```new Message("awd");
assertThat(msg.getContent()).isEqualTo("awd");
```
I nie do końca jest on potrzebny w tym momencie
W teorii nie ma sensu sprawdzać setterów i getterów powinno to wyjść w innych rodzajach testów